### PR TITLE
feat(labels/ui): update label name

### LINKED ISSUE #12578: Cannot rename labels
Cannot rename labels

LabelRenameBug

- [x] Await API update for label name updating: https://github.com/influxdata/influxdb/pull/12585
- [x] Regenerate client types and bump version
- [x] Change `updateLabel` action to take a `Label` and not just properties

### PR DESCRIPTION
**Pull request by 121watts**
Closes #12578

_Briefly describe your proposed changes:_
- imports updates to the client 
- you can now change a label's name 😸

### CODE REVIEW

File: ui/src/shared/utils/labels.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+// Types
+import {Label} from 'src/types/v2/labels'
+import {ILabel as LabelAPI} from '@influxdata/influx'
+
+export const addLabelDefaults = (l: LabelAPI): Label => ({
+  ...l,
+  properties: {
+    ...l.properties,
+    // add default color hex if missing
```
Comment (mavarius) [inline ui/src/shared/utils/labels.ts]: Do we need this comment?

File: ui/src/configuration/components/CreateLabelOverlay.tsx [hunk 2]
```diff
@@ -62,15 +62,15 @@ class CreateLabelOverlay extends Component<Props, State> {
...
               id={label.id}
               name={label.name}
-              description={label.properties.description}
-              color={label.properties.color}
-              onColorChange={this.handleColorChange}
-              onSubmit={this.handleSubmit}
               onCloseModal={onDismiss}
-              onInputChange={this.handleInputChange}
               buttonText="Create Label"
+              onSubmit={this.handleSubmit}
```
Comment (mavarius) [inline ui/src/configuration/components/CreateLabelOverlay.tsx]: Just for the tree?

File: ui/src/shared/utils/labels.ts (hunk 1 above)
Comment (bthesorceror) [reply ui/src/shared/utils/labels.ts]: We do this in the client and don't need to duplicate the work here

File: ui/package-lock.json [hunk 3]
```diff
@@ -3756,7 +3756,7 @@
     },
     "css-color-names": {
       "version": "0.0.4",
-      "resolved": "https://registry.npmjs.org/css-color-names/-/css-color-names-0.0.4.tgz",
+      "resolved": "http://registry.npmjs.org/css-color-names/-/css-color-names-0.0.4.tgz",
```
Comment (OfTheDelmer) [inline ui/package-lock.json]: why downgrade to http?

**Pull request merged**